src/theme/components/button.modifier.ts

```diff
@@ -14,6 +14,7 @@ export const MuiButton: Components<Theme>['MuiButton'] = {
       return {
         ...textB2SemiBold,
         fontWeight: 500,
+        lineHeight: '1.25rem',
```

Comment:
Should this be a statically assigned line height?

What is the line height currently set to; currently inherited as?